tox.ini

```diff
@@ -79,6 +79,16 @@ basepython = python3.7
 deps = mypy
 commands = mypy aea packages tests scripts
 
+[testenv:bandit-main]
+basepython = python3.7
+deps = bandit
+commands = bandit -s B101,B105,B301,B403,B404,B603,B607,B608 -r aea packages scripts
```

Comment:
For the main library and packages I'd like to strive towards only skipping B101. Please @marcofavorito can you look at removing issues around B105,B301,B403,B404,B603,B607 and @Totoual please resolve issues around B608!